requirements/dev.txt

```diff
@@ -5,7 +5,7 @@ ipdb==0.13.8
 black==21.9b0
 django-extensions==3.1.3
 pre-commit==2.15.0
-psycopg2-binary
+# psycopg2-binary
```

Comment:
why did you comment this out. this is used for database access. it should not be commented